### PR TITLE
Episode count on Quick Update

### PR DESCRIPTION
**Pull request by trmcnvn**
Ref: https://kitsu.io/feedback/feature-requests/p/total-episodes-indicator-in-quick-update-tiles

### CODE REVIEW

File: app/components/dashboard/quick-update/item.js [hunk 1]
```diff
@@ -31,9 +31,11 @@ export default Component.extend({
     if (get(this, 'isCompleted')) {
       return get(this, 'intl').t('dashboard.quick-update.media.complete');
     }
-    const type = get(this, 'entry.media.modelType');
-    const num = get(this, 'nextProgress');
-    const text = get(this, 'intl').t('dashboard.quick-update.media.episode', { type, num });
+    const text = get(this, 'intl').t('dashboard.quick-update.media.episode', {
+      type: get(this, 'entry.media.modelType'),
+      num: get(this, 'nextProgress'),
```
Comment (kazzarin) [inline app/components/dashboard/quick-update/item.js:36]: Shouldn't this be `entry.progress`? :thinking:

Comment (trmcnvn) [reply app/components/dashboard/quick-update/item.js:36]: Quick update references the next episode not your progress

Comment (kazzarin) [reply app/components/dashboard/quick-update/item.js:36]: imo it's kinda confusing

It makes it look like I've already watched the 2nd episode and are about to mark the 2nd episode as complete.

Comment (matthewdias) [reply app/components/dashboard/quick-update/item.js:36]: well that was existing functionality on the site. people have asked to change it tho. @JoshFabian ?

**Comment by trmcnvn** [comment]
Yeah this PR hasn't changed any functionality on the progress shown. This
may be best moved to a feature request post if there isn't one already.

**Comment by kazzarin** [comment]
Ah, sorry about that. I found the existing feature request for it.

**Pull request merged**